src/signals/incident-management/components/FilterForm/__tests__/FilterForm.test.js

```diff
@@ -43,10 +44,10 @@ const jsonResponse = {
 const mockResponse = JSON.stringify(jsonResponse);
 
 const formProps = {
-  onClearFilter: () => {},
-  onSaveFilter: () => {},
+  onClearFilter: () => { },
```

Comment:
I see a lot of text editor formatting. Is something changed?

Reply:
Didn't run prettier formatting (only TS formatting). Fixed now.